### PR TITLE
Remove different sound for doors that have same mechanical base.

### PR DESCRIPTION
**Pull request by Heroman3003**
Partial VOREStation-only revert of https://github.com/VOREStation/VOREStation/pull/8658.

 Different paint job or different access restrictions don't make a door sound differently. This makes all doors with same mechanical basis sound the same, approprietly, not based on "what mapped in color they use". So all standard doors now again have same base sounds for opening and closing, as they all share base appearance.

Doesn't affect doors with actually unique appearances, such as pure glass, vault, voidcraft, hatches, highsecurity, doubles, etc. Would also have ignored doors made out of unique materials but those didn't have any sound given to them in the first place, so.

### CODE REVIEW

**Comment by Rykka-Stormheart** [comment]
Am against this getting in.

**Comment by Rykka-Stormheart** [comment]
You also blanket reverted the base airlock sound changes, too.

Doubly against this.

File: code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -39,9 +39,9 @@
 	var/secured_wires = 0
 	var/datum/wires/airlock/wires = null
 
-	var/open_sound_powered = 'sound/machines/door/covert1o.ogg'
```
Comment (Rykka-Stormheart) [inline code/game/machinery/doors/airlock.dm:42]: Why are these reverted?

File: code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -39,9 +39,9 @@
 	var/secured_wires = 0
 	var/datum/wires/airlock/wires = null
 
-	var/open_sound_powered = 'sound/machines/door/covert1o.ogg'
+	var/open_sound_powered = 'sound/machines/door/airlock.ogg'
 	var/open_sound_unpowered = 'sound/machines/airlockforced.ogg'
-	var/close_sound_powered = 'sound/machines/door/covert1c.ogg'
```
Comment (Rykka-Stormheart) [inline code/game/machinery/doors/airlock.dm:44]: Ditto.

**Comment by Rykka-Stormheart** [comment]
I still say this should be left unmerged until release is on live for a few days to gather feedback. Reverting this before the changes are even in is hugely kneejerk, imo.

**Comment by Heroman3003** [comment]
The reason I am doing this PR is because of how previously this same change was turned down to be redone in the different way, and only reason it got merged this time is because maintainer merging it didn't notice the change due to misleading description of the PR. Decision that this should be done this way has been done months ago.

**Comment by Rykka-Stormheart** [comment]
> The reason I am doing this PR is because of how previously this same change was turned down to be redone in the different way, and only reason it got merged this time is because maintainer merging it didn't notice the change due to misleading description of the PR. Decision that this should be done this way has been done months ago.

That's a rather snide attempt at saying "you lied". I did not lie, the PR's I pulled changes from were linked upfront in the PR.

File: code/game/machinery/doors/airlock.dm [hunk 3]
```diff
@@ -39,9 +39,9 @@
 	var/secured_wires = 0
 	var/datum/wires/airlock/wires = null
 
-	var/open_sound_powered = 'sound/machines/door/covert1o.ogg'
+	var/open_sound_powered = 'sound/machines/airlock.ogg'
 	var/open_sound_unpowered = 'sound/machines/airlockforced.ogg'
-	var/close_sound_powered = 'sound/machines/door/covert1c.ogg'
+	var/close_sound_powered = 'sound/machines/airlockclose.ogg'
```
Comment (Novacat) [inline code/game/machinery/doors/airlock.dm:44]: I would like this to be reverted back.

**Comment by Heroman3003** [comment]
Links in and of themselves tell you nothing, and 99% of the time are only posted as additions to list of changes. Except list of changes in said PR did not mention changes from the links themselves. I didn't say it was intentionally misleading.

As for the revert of the base door sound, part of the request.

File: code/game/machinery/doors/airlock.dm (hunk 3 above)
Comment (Rykka-Stormheart) [reply code/game/machinery/doors/airlock.dm:44]: I'd suggest using hall1c and hall1o for close and open sounds respectively.

They fit well for the doors.

**Comment by Rykka-Stormheart** [comment]
I'd still argue that this wait a few days for user feedback and 'min feedback before we revert it fully.

Is it okay to ask for staff discussion on this?

**Pull request closed without merging**